src/qoi/include/grins/qoi_base.h

```diff
@@ -78,6 +78,9 @@ namespace GRINS
 
     virtual void init_context( AssemblyContext& /*context*/ ){}
 
+    //! Reinitialize QoI
+    virtual void reinit(MultiphysicsSystem & /*system*/) const {}
```

Comment:
Why is this a `const` method? By definition we're changing underlying things in the `QoIBase` object, so I don't think it should be `const`.